Libplanet.Explorer/GraphTypes/IValueType.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Bencodex;
+using GraphQL.Language.AST;
+using GraphQL.Types;
+
+namespace Libplanet.Explorer.GraphTypes
+{
+    public class IValueType : StringGraphType
+    {
+        private static readonly Codec _codec = new ();
+
+        public IValueType()
+        {
+            Name = "IValue";
+        }
+
+        public override object? Serialize(object? value)
+        {
+            if (value is Bencodex.Types.IValue iv)
+            {
+                return _codec.Encode(iv);
+            }
+
+            return value;
+        }
+
+        public override object? ParseValue(object? value)
+        {
+            return value switch
+            {
+                null => null,
+                string hex => _codec.Decode(ByteUtil.ParseHex(hex)),
+                _ => throw new ArgumentException(
+                    $"Expected a hexadecimal string but {value}",
+                    nameof(value)
+                ),
+            };
+        }
+
+        public override object? ParseLiteral(IValue? value)
+        {
+            if (value is StringValue)
+            {
+                return ParseValue(value.Value);
+            }
+
+            return null;
```

Comment:
How about throwing errors instead of `null` if unexpected types came?

You can see the `GuidGraphType` as an example.

https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL/Types/Scalars/GuidGraphType.cs

```suggestion
            return ThrowLiteralConversionError(value);
```